provider/ec2/environ.go

```diff
@@ -71,11 +70,17 @@ const (
 )
 
 var (
-	// Use shortAttempt to poll for short-term events or for retrying API calls.
-	// TODO(katco): 2016-08-09: lp:1611427
-	shortAttempt = utils.AttemptStrategy{
-		Total: 5 * time.Second,
-		Delay: 200 * time.Millisecond,
+	// Use shortRetryStrategy to poll for short-term events or for retrying API calls.
+	shortRetryStrategy = retry.CallArgs{
+		Clock:       clock.WallClock,
+		MaxDuration: 5 * time.Second,
+		Delay:       200 * time.Millisecond,
+	}
+	// Use longRetryStrategy to poll for results that are eventually consistent
+	longRetryStrategy = retry.CallArgs{
```

Comment:
I'd be inclined to just create this at its only usage site.

Reply:
Done